src/script/standard.cpp

```diff
@@ -89,21 +89,55 @@ static constexpr bool IsSmallInteger(opcodetype opcode)
     return opcode >= OP_1 && opcode <= OP_16;
 }
 
+static constexpr bool IsPushdataOp(opcodetype opcode)
+{
+    return opcode > OP_FALSE && opcode <= OP_PUSHDATA4;
+}
+
+/** Valid number of keys in a multisig are [1,20]. */
+static constexpr bool IsValidMultisigKeyCount(unsigned int n_keys)
+{
+    return n_keys > 0 && n_keys <= MAX_PUBKEYS_PER_MULTISIG;
```

Comment:
0 is a valid value for number of keys, so this should be

```suggestion
    return n_keys >= 0 && n_keys <= MAX_PUBKEYS_PER_MULTISIG;
```

Reply:
I don't think it's valid according to policy: `IsSmallInteger` currently requires the number of keys to be `>= OP_1`

Reply:
Policy is not validity...

Reply:
@luke-jr i think here the term was used to refer to policy, as `MatchMultisig` is only used to define standardness. What i meant is that i don't think i changed behaviour here (`IsSmallInteger` was already required for both counts).

Reply:
Maybe the function should get renamed for clarity :)

`IsAcceptableMultisigKeyCount` or such

Reply:
Ah, `IsSmallInteger` also disallows 0, so behavior is not changing in this area.